app/fruits/[fruitName]/page.js

```diff
@@ -30,13 +29,37 @@ export function generateMetadata({ params }) {
 export const dynamic = 'force-dynamic';
 
 export default function FruitPage({ params }) {
-  const singleFruit = fruits.find((fruit) => {
-    return fruit.name.toLowerCase() === params.fruitName;
-  });
+  const singleFruit = getFruitByName(params.fruitName);
 
   if (!singleFruit) {
     notFound();
   }
 
-  return <Fruit fruit={singleFruit} />;
+  let fruitNotesCookie = getCookieByName('fruitNotes');
+
+  if (!Array.isArray(fruitNotesCookie)) {
+    fruitNotesCookie = [];
+  }
+
+  const singleFruitNote = fruitNotesCookie.find(
+    (fruitNote) => fruitNote.id === singleFruit.id,
+  );
+
+  return (
+    <>
+      <h1>{params.fruitName}</h1>
+      <p
```

Comment:
we don't really use `p` elements in our apps (adds extra margin, which is often unwanted)